src/modules/SX128x/SX128x.cpp

```diff
@@ -529,7 +529,12 @@ int16_t SX128x::startReceive(uint16_t timeout) {
   }
 
   // set DIO mapping
-  int16_t state = setDioIrqParams(RADIOLIB_SX128X_IRQ_RX_DONE | RADIOLIB_SX128X_IRQ_RX_TX_TIMEOUT | RADIOLIB_SX128X_IRQ_CRC_ERROR | RADIOLIB_SX128X_IRQ_HEADER_ERROR, RADIOLIB_SX128X_IRQ_RX_DONE);
+  uint16_t mask = RADIOLIB_SX128X_IRQ_RX_DONE;
+
+  if(timeout != RADIOLIB_SX128X_RX_TIMEOUT_INF)
```

Comment:
Please use the coding style guideline, including braces around single-line statements, thanks!

Reply:
Sure, force of habit, added braces :-)